README.md

```diff
@@ -263,6 +257,119 @@ This would create a structure such as:
+1. Run `medic-conf csv-to-docs upload-docs create-users` 
+	1. This will generate the contacts, places, and users associated to those contacts. The users are placed into a users.csv file in your working directory. Then upload the json docs creating your data and creating users associated.
+
+
+Here is a example of how the three csvs need to be configured to setup a user linked to existing place and contact.  
+
+place.health_center.csv
+
+```
+reference_id:excluded,parent:place WHERE reference_id=COL_VAL,is_name_generated,name,reported_date:timestamp
+health_center_1,district_1,FALSE,HC1,1544031155715
+```
+Generated json doc for the health center
+```
+{
+  "type": "health_center",
+  "parent": {
+    "type": "district_hospital",
+    "parent": "",
+    "is_name_generated": "false",
+    "name": "District1",
+    "external_id": "",
+    "notes": "",
+    "geolocation": "",
+    "reported_date": 1544031155715,
+    "_id": "e8f9739a-5d37-5b1e-be3c-a571b2c2409b"
+  },
+  "is_name_generated": "FALSE",
+  "name": "HC1",
+  "reported_date": 1544031155715,
+  "_id": "8606a91a-f454-56e3-a089-0b686af3c6b7"
+}
+```
+
+person.csv
+
+```
+reference_id:excluded,parent:place WHERE reference_id=COL_VAL,name,phone,sex,role,reported_date,patient_id
+p_hc1,health_center_1,Bob Johnson 1,+16143291527,male,manager,1552494835669,60951
+p_hc2,health_center_1,Bob Johnson 2,+16143291528,male,manager,1552494835669,60951
+
+```
+Generated json doc for the person
+```
+{
+  "type": "person",
+  "parent": {
+    "type": "health_center",
+    "parent": {
+      "type": "district_hospital",
+      "parent": "",
+      "is_name_generated": "false",
+      "name": "District1",
+      "external_id": "",
+      "notes": "",
+      "geolocation": "",
+      "reported_date": 1544031155715,
+      "_id": "e8f9739a-5d37-5b1e-be3c-a571b2c2409b"
+    },
+    "is_name_generated": "FALSE",
+    "name": "HC1",
+    "reported_date": 1544031155715,
+    "_id": "8606a91a-f454-56e3-a089-0b686af3c6b7"
+  },
+  "name": "Bob Johnson 1",
+  "phone": "+16143291527",
+  "sex": "male",
+  "role": "manager",
+  "reported_date": "1552494835669",
+  "patient_id": "60951",
+  "_id": "65c52076-84c5-53a2-baca-88e6ec6e0875"
+}
+
+```
+
+users.csv 
+```
+username,password,roles,phone,contact:person WHERE reference_id=COL_VAL,place:GET _id OF place WHERE reference_id=COL_VAL
+ac1,Secret_1,district_admin:red1,+123456789,p_hc1,health_center_1
+ac2,Secret_1,district_admin:supervisor,+123456789,p_hc2,health_center_1
+ac3,Secret_1,district_admin,+123456789,p_hc3,health_center_1
+ac4,Secret_1,district_admin,+123456789,p_hc4,health_center_1
+
+```
+Generating a new users.csv with the person and place set to the uuids for the above person and place
```

Comment:
"This will generate the `users.csv` file in the working directory which is used by the `create-users` action. The contact and place fields should be resolved to the actual UUIDs." ?